tests/ConfigCatClientTest.php

```diff
@@ -429,60 +473,62 @@ public function testHooks()
+        $client = new ConfigCatClient('testEvalDetails-789012/1234567890123456789012', [ClientOptions::CUSTOM_HANDLER => new MockHandler([
             new Response(200, [], Utils::formatConfigWithRules()),
         ])]);
 
         $details = $client->getValueDetails('key', '', new User('test@test1.com'));
 
         $this->assertEquals('fake1', $details->getValue());
         $this->assertEquals('id1', $details->getVariationId());
-        $this->assertNull($details->getError());
+        $this->assertNull($details->getErrorMessage());
         $this->assertEquals('key', $details->getKey());
         $this->assertEquals('test@test1.com', $details->getUser()->getIdentifier());
-        $this->assertEquals('Identifier', $details->getMatchedEvaluationRule()['a']);
-        $this->assertEquals('@test1.com', $details->getMatchedEvaluationRule()['c']);
-        $this->assertEquals(2, $details->getMatchedEvaluationRule()['t']);
-        $this->assertNull($details->getMatchedEvaluationPercentageRule());
+        $condition = $details->getMatchedTargetingRule()['c'][0]['u'];
+        $this->assertEquals('Identifier', $condition['a']);
+        $this->assertEquals('@test1.com', $condition['l'][0]);
+        $this->assertEquals(2, $condition['c']);
+        $this->assertNull($details->getMatchedPercentageOption());
         $this->assertTrue($details->getFetchTimeUnixMilliseconds() > 0);
         $this->assertFalse($details->isDefaultValue());
     }
 
     public function testEvalDetailsNonExistentFlag()
     {
-        $client = new ConfigCatClient('testEvalDetails', [ClientOptions::CUSTOM_HANDLER => new MockHandler([
+        $client = new ConfigCatClient('testEvalDetails-789012/1234567890123456789012', [ClientOptions::CUSTOM_HANDLER => new MockHandler([
             new Response(200, [], Utils::formatConfigWithRules()),
         ])]);
 
         $details = $client->getValueDetails('non-existent', '', new User('test@test1.com'));
 
         $this->assertEquals('', $details->getValue());
         $this->assertEquals('', $details->getVariationId());
-        $this->assertNotNull($details->getError());
+        $this->assertNotNull($details->getErrorMessage());
         $this->assertEquals('non-existent', $details->getKey());
         $this->assertEquals('test@test1.com', $details->getUser()->getIdentifier());
-        $this->assertNull($details->getMatchedEvaluationRule());
-        $this->assertNull($details->getMatchedEvaluationPercentageRule());
+        $this->assertNull($details->getMatchedTargetingRule());
+        $this->assertNull($details->getMatchedPercentageOption());
         $this->assertTrue($details->isDefaultValue());
     }
 
     public function testEvalDetailsHook()
     {
-        $client = new ConfigCatClient('testEvalDetailsHook', [ClientOptions::CUSTOM_HANDLER => new MockHandler([
+        $client = new ConfigCatClient('testEvalDetailsHook-12/1234567890123456789012', [ClientOptions::CUSTOM_HANDLER => new MockHandler([
```

Comment:
I would use a TEST_SDK_KEY const here.

Reply:
These tests are running parallel, these unique SDK keys are needed because of the singleton clients